SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ExpectedViewDelta.cpp

```diff
@@ -96,6 +96,26 @@ bool ExpectedViewDelta::Compare(const ViewDelta& Other)
 		return false;
 	}
 
+	// Calling GetConnectionStatus() when we are still connected will result in failing an assertion.
```

Comment:
weird- why? Shouldn't it just return "connected"?

Reply:
@samiwh  - I was wondering that as well. wouldn't it make more sense to just return the status?

Reply:
Checked with him. It's because the view delta only has the connection status & disconnect messafe if it was part of the op list in that tick. Gonna rename the methods to something like 
`HasConnectionStatusChanged`, `GetConnectionStatusChange`, `GetConnectionStatusMessage` to make that a bit clearer

Reply:
Sounds good. Should probably also change the comment here too to match.